### ISSUE TITLE
When bar is hidden touch gestures don't work

### ISSUE THREAD

**Issue by JamesSwift** [mlutfy/hidetopbar#218]
With the bar visible, I can:
- swipe down from the centre to reveal notifications
- swipe down from the right corner to reveal the settings menu

With the bar hidden, neither of those gestures work.

**Comment by BPaden** [comment]
You also can't view notifications with the keyboard shortcut (super+v). You have to hit super to open the overview first and then hit super+v, which is a little awkward. Not sure if this has the same underlying cause or if it should be its own issue.

**Comment by tuxor1337** [comment]
Yes, it might be related. You could try the workaround (comment out a single line of code) described here: https://github.com/mlutfy/hidetopbar/issues/228

**Comment by BPaden** [comment]
Thanks, that workaround works. And sorry for the noise, my eyes completely skipped over that issue.
